src/sentry/static/sentry/app/components/links/linkWithConfirmation.tsx

```diff
@@ -4,32 +4,52 @@ import classNames from 'classnames';
 
 import Confirm from 'app/components/confirm';
 
+type Props = {
+  disabled?: boolean;
+  message: React.ReactNode;
+  title: string | undefined;
+  onConfirm: () => void;
+} & React.HTMLAttributes<HTMLAnchorElement>;
```

Comment:
Should this take the props of `Confirm`? We don't pass any other props to the anchor

Reply:
Fixed to take in props of `Confirm`.